portafly/doc/ux_specs/audience/accounts/Listing.md

```diff
@@ -0,0 +1,56 @@
+# Audience > Accounts > Listing
+
+* [Mockups](https://marvelapp.com/55343de/screen/67190788)
+* [Mockups JIRA ticket](https://issues.redhat.com/browse/THREESCALE-4725)
+* [Parent JIRA ticket](https://issues.redhat.com/browse/THREESCALE-4724)
+
+##### Page header
+* Create button
+  * Icon is `plus-circle`
+* Export button
+  * Icon is `pficon-save`
+
+##### Toolbar ([specs](../../global_components/toolbar.md))
+* Bulk actions:
+  * Send email
+  * Change account
+  * Change state
+* Filters:
+  * Group/org > text input ([PF specs](https://www.patternfly.org/v4/documentation/react/components/inputgroup#with-dropdown))
+  * Admin > text input ([PF specs](https://www.patternfly.org/v4/documentation/react/components/inputgroup#with-dropdown))
+  * State > Dropdown menu with multi-select checklist ([PF specs](https://www.patternfly.org/v4/documentation/react/components/select#checkbox-input))
+     * "Approved"
+     * "Pending"
+     * "Rejected"
+     * "Suspended"
+  <!-- * Plan > Dropdown menu with multi-select checklist ([PF specs](https://www.patternfly.org/v4/documentation/react/components/select#checkbox-input)) -- with filter when many options are available ([PF specs](https://www.patternfly.org/v4/documentation/react/components/select#grouped-checkbox-input-with-filtering))
+     * list of current plans -->
+
+##### Table ([PF specs](https://www.patternfly.org/v4/documentation/react/components/table))
+* Table header:
```

Comment:
Isn't a column for `Plan` missing? (which depends on some payment condition if I recall correctly)

Reply:
@josemigallas  we're discussing removing account plans for 3.x

Reply:
as per @thomasmaas comment "account plans and service plans won’t make it to 3.0 for sure"